lib/server/services/lhcFill/getAllLhcFillsOrFail.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+const { getLhcFillByNumberOrFail } = require('./getLhcFillByNumberOrFail.js');
+
+/**
+ * Returns the list of lhcFill ids extracted from a comma separated (CSV) list of lhcFill numbers
+ *
+ * @param {number[]} [lhcFills] the CSV list of lhcFill numbers
+ * @return {Promise<null|number[]>} the list of lhcFill ids
```

Comment:
When you return an array you should avoid returning nullable array. This may lead to weird bugs (especially when using js without proper IDE hints), because we may forgot to check for null.